README.md

```diff
@@ -93,23 +87,6 @@ Once configured, check the [Logs tab](https://docs.viam.com/program/debug/) of y
 ```
 ___
 
-## Latest Version Installation
```

Comment:
This is likely to cause more confusion than help -- no one should be doing this unless they really know what they're doing.